build/ci/.azure-devops-android-tests.yml

```diff
@@ -48,6 +48,11 @@ jobs:
 
   - template: templates/gitversion.yml
 
+  - script: |
+      cd $(build.sourcesdirectory)/src/SamplesApp/SamplesApp.netcoremobile
+      dotnet workload restore
+    displayName: Workload restore
```

Comment:
This should not be needed if we have run uno.check. Doing this will likely result in build failures down the road.

Why did you need to add it? We may need to change uno.check instead.

Reply:
@jeromelaban It's all about maintenance hassle of manifests, especially that now we are just using .NET 9 SDK ad building for both net8.0/net9.0 TFMs. `dotnet workload restore` will always do the right thing and just install the right workloads. Otherwise, it's a nightmare to maintain different manifests and having to create ones that work with .NET 9 SDK but for net8.0 builds, etc. And I think if we will later build both net8.0 and net9.0 in the same job, we will need two workload installation. But `dotnet workload restore` should get us working almost with zero maintenance cost. It hopefully shouldn't result in build failures, but we'll see.